app/config/AppStartupJobs.scala

```diff
@@ -0,0 +1,140 @@
+                         actorSystem: ActorSystem
+                       )(implicit ec: ExecutionContext) {
+
+  actorSystem.scheduler.scheduleOnce(FiniteDuration(1, TimeUnit.MINUTES)) {
+    appStartupJobs.runEverythingOnStartUp()
+  }
+}
+
+class AppStartupJobsImpl @Inject()(val config: Configuration,
+                                    val mongo: MongoComponent
+                                  )(implicit val ec: ExecutionContext) extends AppStartupJobs
+
+trait AppStartupJobs extends Logging {
+
+  implicit val ec: ExecutionContext
+  val mongo: MongoComponent
+
+  val processReadyCalsReqCollection: MongoCollection[ProcessReadyCalculationRequest] =
+    CollectionFactory.collection(mongo.database, "bulk-calculation", ProcessReadyCalculationRequest.formats)
+
+  val processedBulkCalsReqCollection: MongoCollection[ProcessedBulkCalculationRequest] =
+    CollectionFactory.collection(mongo.database, "bulk-calculation", ProcessedBulkCalculationRequest.formats)
+
+  private def logParentsMissingCreatedAtAndChildren(): Future[Unit] = {
+    val filter = Filters.and(
+      Filters.eq("isParent", true),
+      Filters.exists("createdAt", false)
+    )
+
+    processedBulkCalsReqCollection.find(filter).toFuture().flatMap { parents =>
+      logger.info(s"[runEverythingOnStartUp] Found ${parents.size} parent documents missing createdAt")
+
+      val childCountFutures = parents.map { parent =>
+        val parentId = parent._id
+
+        val childFilter = Filters.and(
+          Filters.eq("isChild", true),
+          Filters.eq("bulkId", parentId)
+        )
+
+        processReadyCalsReqCollection.countDocuments(childFilter).toFuture().map { childCount =>
+          logger.info(
+            s"[runEverythingOnStartUp] Parent uploadReference=$parentId has $childCount children."
+          )
+        }
+      }
+
+      Future.sequence(childCountFutures).map(_ => ())
+    }.recover {
+      case ex =>
+        logger.error("[runEverythingOnStartUp] Failed to fetch parents missing createdAt", ex)
+    }
+  }
+
+
+  def runEverythingOnStartUp(): Future[Unit] = {
+    logger.info("[runEverythingOnStartUp] Running Startup Jobs...")
+
+    val missingCreatedAtFilter = Filters.and(
+      Filters.eq("isChild", true),
+      Filters.exists("createdAt", false)
+    )
+
+    val incompleteParentsFilter = Filters.and(
+      Filters.eq("isParent", true),
+      Filters.eq("complete", false)
+    )
+
+    val childCount = logCount(
+      collection = processReadyCalsReqCollection,
+      filter = missingCreatedAtFilter,
+      description = "child documents missing createdAt"
+    )(ExecutionContext.global)
+
+    val parentCount = logCount(
+      collection = processedBulkCalsReqCollection,
+      filter = incompleteParentsFilter,
+      description = "incomplete parent documents (complete = false)"
+    )(ExecutionContext.global)
+
+    val parentMissingCreatedAtAndChildren = logParentsMissingCreatedAtAndChildren()
+
+    for {
```

Comment:
I believe these will run in parallel, so each instance will run all 3 queries as 1 minute after deployment
If there's any risk of these queries being expensive then we may want to make them sequential